client/app/scripts/superdesk-dictionaries/dictionaries.js

```diff
@@ -63,7 +63,15 @@
         };
 
         this.update = function update(dictionary, data, success, error) {
-            return api.save('dictionaries', dictionary, data).then(success, error);
+            var sendData = {};
+            angular.forEach(data, function(val, key) {
+                if (key !== 'content') {
+                    sendData[key] = val === null? val: val.toString();
+                } else {
+                    sendData[key] = val;
+                }
```

Comment:
OK, I guess we simply read things differently, can't argue that.
(FWIW, I mentally "see" a nested _if-then-else_ inside the outer `if`, while the compact form reads just as a single _if-then-else_).